meta/documents/user_guide/de/initial_setup.md

```diff
@@ -79,33 +79,31 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
 
 1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >> `Artikel bearbeiten`.
 
-2. Suchen Sie nach Artikeln und öffnen Sie sie
+2.  Suchen Sie nach Artikeln und öffnen Sie sie **`Stellen Sie sicher, dass auf dem Artikel die **`Hauptvariante`** steht`** siehe Bild unten
 
-3. **Klicken Sie für jedes Element** auf `Varianten` und öffnen Sie diese.
+    ![item main variation](../../../images/de/item_main_variation.png)
 
-4. **Für jede Variante**:
-
-    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+3. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
 
         ![variation active](../../../images/de/variation_active_field.png)

```

Comment:
This doesn't show up in the rich view

Reply:
Yeah this image seems to have gone missing somewhere!

Reply:
@jathmel I think you need to remove some of the indentations before this line, in order for the image to get rendered.